README.md

```diff
@@ -9,7 +9,7 @@ Files <a href="https://github.com/files-community/files-uwp/blob/master/CODE_OF_
 <img src="Files/Assets/FilesHome.png" width="600px"><br>
 ## Download Files 
 - <a href="https://www.microsoft.com/store/apps/9NGHP3DX8HDX">Microsoft Store</a>
-- <a href="https://github.com/files-community/files-uwp/releases/download/v0.14/FilesUWP-0.14.Msixbundle">GitHub</a>
+- <a href="https://github.com/files-community/Files/releases/download/v0.23.1/Files-0.23.1.Msixbundle">GitHub</a>
```

Comment:
```suggestion
- <a href="https://github.com/files-community/Files/releases">GitHub</a>
```

Perhaps the download link should just point to the release page, this way we won't have to constantly update the readme file everytime we release an new version of Files.

Reply:
People downloading from github should be smart enough to find the msix download link right? xD